cypress/e2e/tests/navigation/side-nav/main-side-menu.spec.ts

```diff
@@ -39,22 +44,6 @@ describe('Side Menu: main', () => {
     BurgerMenuPo.checkTooltipOn();
   });
 
-  it('Check that the local cluster badge matches LCL, custom badge should match EX', { tags: ['@adminUser'] }, () => {
```

Comment:
Can you create an issue to bring this back in at some point? It tested 
- abbreviations
- clicking a cluster menu item takes you to a specific cluster
- custom cluster label
so there's a good bit of value there

Reply:
It should also run for standardUser, who it think we set up with access to the local cluster

Reply:
@scures has already created one https://github.com/rancher/dashboard/issues/9658

Reply:
That's for unit testing the function that provides the abbreviation, this is e2e testing. There's crossover with a couple, but not the middle point